src/generators/LROGenerator.ts

```diff
@@ -11,25 +11,21 @@ export async function generateLROFiles(
     return;
   }
 
-  const lroFiles = [
-    { name: "bodyPollingStrategy", file: "./src/lro/bodyPollingStrategy.ts" },
-    { name: "lroPoller", file: "./src/lro/lroPoller.ts" },
-    { name: "models", file: "./src/lro/models.ts" },
-    { name: "operation", file: "./src/lro/operation.ts" }
-  ];
+  const lroDir = "./src/lro";
+  const lroFiles = await promises.readdir(lroDir);
```

Comment:
Thanks for catching this, I've fixed it. I had to jump a bit more since autorest.core would pickup the already build autorest.typescript and __dirname would be under /dist